### DISCUSSION TITLE
Why Plastic Vertex names with space + double underscore?

### DISCUSSION THREAD

**Discussion by gab3d** [opentoonz/opentoonz#6463] [Q&A]
### --Issue Summary--
Currently Plastic Tool vertices names have "a space + a double underscore" (" __") added to its name.
That's a strange thing, and I don't know if there's an actual functional reason behind it or it's just that it's been that way since Harlequin days.

If there's no functional reason behind it, in my opinion, it could be a good idea to replace it by just an underscore.

### --Steps to reproduce--
1. Create a level
2. Activate the Plastic Tool
3. Create a Mesh
4. Create some Vertices

### --Actual Results--
Vertices names in the form: "Vertex __6", "Vertex __7", etc.

### --Expected Results--
Vertices names in the form: "Vertex_6", "Vertex_7", etc. ??

### --System Information--
+ OpenToonz Version:
1.2.1 RC

+ Operating System:
all

**Comment by DarrenTAnims** [comment]
That sounds like a sensible change to me. It's difficult enough reading names in various control when they get long, so keeping them as short as possible is a good idea. And there's no visual reason to not do so.
